src/validation.cpp

```diff
@@ -2810,6 +2810,18 @@ bool Chainstate::ConnectTip(BlockValidationState& state, CBlockIndex* pindexNew,
              Ticks<SecondsDouble>(time_total),
              Ticks<MillisecondsDouble>(time_total) / num_blocks_total);
 
+    // If we are the background validation chainstate, check to see if we are done
+    // validating the snapshot (i.e. our tip has reached the snapshot's base block).
+    if (this != &m_chainman.ActiveChainstate()) {
+        // This call may set `m_disabled`, which is referenced immediately afterwards in
+        // ActivateBestChain.
```

Comment:
In commit "validation: add ChainMan logic for completing UTXO snapshot validation" (https://github.com/bitcoin/bitcoin/commit/e17d51768bcf5f7791fdf7d0b2ee4b11cc64d922)

Would add "so it will stop connecting new blocks" to make it clearer what m_disabled is used for.